docs/repo-docs/crafting-your-repository/configuring-tasks.mdx

```diff
@@ -172,7 +172,7 @@ Some tasks may not have any dependencies. For example, a task for finding typos
   tasks are properly configured first.
 </Callout>
 
-The `outputs` key tells Turborepo **files and directories** it should cache when the task has successfully completed. **Without this key defined, Turborepo will not cache any files. Hitting cache on subsequent runs will not restore any file outputs.**
+The `outputs` key tells Turborepo **files and directories** it should cache when the task has successfully completed.
```

Comment:
We need to keep this information. It's important that folks know that no artifacts will be cached.